src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,19 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos = [] }) => (
```

Comment:
```suggestion
export const TodoList: React.FC<Props> = ({ todos }) => (
```
todos is required props aaccording to your Props type